modules/dbnd/src/dbnd/_core/cli/cmd_run.py

```diff
@@ -296,11 +296,15 @@ def run(
             print_help(ctx, task_cls)
             return
 
-        return context.dbnd_run_task(
+        dbnd_run = context.dbnd_run_task(
             task_or_task_name=task_name,
             run_uid=run_driver,
             scheduled_run_info=scheduled_run_info,
         )
+        if open_web_tab:
```

Comment:
we should do it in run tracker,  (RunTracker) on run start
we can bypass the value via configuration 
[run] open_web_tracker_in_browser.      (object RunConfig) 

otherwise --> this will open tracker only when run ends.

Reply:
I think I fixed it

Reply:
However as I said on slack - I'm not sure how to check if it works